test/test.js

```diff
@@ -292,4 +305,114 @@ describe('apostrophe-review-and-deploy', function() {
     });
   });
 
+  it('prepare a second export', function() {
```

Comment:
I'd wrap these 2 cases (to line 344) in a `describe()` function, since the second test depend on the value of `exported` from the one before it. This will ensure that if the test cases get reordered in the future, these 2 cases are much more likely to stay in the correct order.

I think there's a similar test case order dependency above, which could probably get this same treatment.

Reply:
Actually, maybe it's `describe('rollbacks')` and it goes to the end of the file, with a comment about maintaining the order within the describe. Thoughts?